src/pickley/bstrap.py

```diff
@@ -160,8 +159,15 @@ def find_base(base):
     abort(f"Make sure '{candidates[0]}' is writeable.")
 
 
-def get_latest_version(package_name):
-    data = http_get(f"https://pypi.org/pypi/{package_name}/json")
+def get_latest_pickley_version(mirror):
+    if not mirror:
+        mirror = "https://pypi.org/simple"
+
+    # This is a temporary measure, eventuall we'll use `uv describe` for this
+    url = os.path.dirname(mirror)
+    url = f"{url}/pypi/pickley/json"
+    print(f"Querying {url}")
```

Comment:
print?

Reply:
The boostrap script simply `print()` to stdout...
Way back in 2017 it was due to how one of the tools using this was capturing things, I believe I could nowadays change that to at least output to stderr yes (but would have to confirm)